### PR TITLE
Improved code so that clicking on thumbnail of image should open it in a modal (#23301r3)

### PR DESCRIPTION
**Pull request by azkyakhan**
…

### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/157401922-029ad329-6391-4927-98d9-7c98c980adde.mp4

**Comment by dt2patel** [comment]
@ymaheshwari1 does this look good to you?

File: src/views/PurchaseOrderDetail.vue [hunk 1]
```diff
@@ -154,6 +155,13 @@ export default defineComponent({
     })
   },
   methods: {
+    async image(imageUrl: string, productName: string) {
```
Comment (adityasharma7) [inline src/views/PurchaseOrderDetail.vue:158]:
```suggestion
    async openImage(imageUrl: string, productName: string) {
```

File: src/views/PurchaseOrderDetail.vue [hunk 2]
```diff
@@ -154,6 +155,13 @@ export default defineComponent({
...
+      const createmodal = await modalController.create({
```
Comment (adityasharma7) [inline src/views/PurchaseOrderDetail.vue:159]:
```suggestion
      const imageModal = await modalController.create({
```

File: src/views/PurchaseOrderDetail.vue [hunk 3]
```diff
@@ -154,6 +155,13 @@ export default defineComponent({
...
+        component: ImageModal,
+        componentProps: {imageUrl , productName}
```
Comment (adityasharma7) [inline src/views/PurchaseOrderDetail.vue:161]:
```suggestion
        componentProps: { imageUrl , productName }
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

File: src/views/PurchaseOrderDetail.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/PurchaseOrderDetail.vue]: I have replaced method name from `image` with `openImage`.

File: src/views/PurchaseOrderDetail.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/PurchaseOrderDetail.vue]: I have replaced `createmodal` with `imageModal`.

File: src/views/PurchaseOrderDetail.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/PurchaseOrderDetail.vue]: I have added space here.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**